### ISSUE TITLE
[Vancouver] Engineering Team Lead @ Thinkific

### ISSUE THREAD

**Issue by leonardofaria** [backend-br/vagas#1409]
## Descrição - About the position

**Are you an experienced software engineer with experience mentoring, managing a team and positioning them for success? For you, it’s not just about writing simple and beautiful code, but also about communicating with stakeholders and making your team successful. You’re looking for a big challenge that holds big opportunities—and you’ll find that at Thinkific!**

As an **Engineering Team Lead**, you will work closely with our engineering, product design and product management teams to guide and mentor a team of 4-7 people to deliver platform features and enhancements into the Thinkific stack. You’ll work with amazing engineers who are passionate about creating great experiences for our users, while balancing technical and project needs. We want to build quickly, with high-quality code and paradigms that bring joy and open up possibilities in the Thinkific platform. You’ll be responsible for helping the team plan and organize their work, while supporting successful execution through mentorship and coaching opportunities.

**In this role, you will:**

*   Keep up to date with the latest trends and technologies in software, in order to ensure that best practices for quality software development and testing methodologies are followed 
*   Work with product managers, project leads and engineers to ensure that business requirements have been turned into technical requirements with clear technical specifications, and work with your team to identify and implement the specifications
*   Act as a mentor to other engineers and ensure they have everything they need to do their best work
*   Foster an open, collaborative, and inclusive environment for the team
*   Engage in recurring 1-on-1s with team members and support their development through regular coaching and mentorship
*   Prepare and execute annual performance reviews and 6 month check-ins for team members
*   Work as an individual contributor using the Ruby on Rails and Golang software stack
*   Support engineers effectively by identifying and unblocking the team, continually looking for ways to improve team efficiency and productivity
*   Review implementation plans with the team 
*   Work with Project Leads to establish timelines for the completion of the project and facilitate technical estimation by either doing technical estimation directly or supporting a team member’s effort
*   Triage incoming bugs and tickets for the team and ensure triaged bugs get into sprints
*   Be on call for support as needed—these are rotating, low-impact shifts

**To be successful in this role, you must:**

*   Have a minimum of 5+ years in a full stack role, with at least 2+ years leading others
*   Demonstrate accountability for technical projects in the past and understand how to make projects successful
*   Have a practical understanding of Web APIs, HTML, CSS and JavaScript
*   Be proficient with Ruby and Ruby on Rails applications
*   Showcase experience with CSS processors such as PostCSS, Sass, or Less
*   Have experience with ES6 and at least one modern JavaScript library/framework such as React, Ember, Angular, Vue, etc.
*   Be responsible for giving feedback to others to help improve their performance (and be receptive to receiving feedback from others)
*   Demonstrate proficiency designing and implementing RESTful services
*   Believe that writing tests as part of a feature is not just a nice to have, but is necessary
*   Demonstrate experience using Git and related tools like GitHub, GitLab, etc
*   Have experience with relational databases, SQL and engines like PostgreSQL, MySQL
*   Have experience with web application architecture and web server technologies like Apache, Nginx, ISS, etc.

## Benefícios - Perks

*   Be part of a collaborative, driven team focused on building best-in-class software for one of the fastest-growing companies in Vancouver
*   We offer competitive salaries, health/dental/vision benefits, and our Employee and Family Assistance Program helps to support your well-being
*   We have an open vacation policy and flexible work environment… because sometimes you do your best work in slippers ;)
*   Our office is stocked with snacks, coffee, and the occasional Friday beer and cider to keep you fuelled
*   We're dog-friendly and have many furry friends sitting nearby, ready to be cuddled
*   We get together often for fun company events (we’ve flown to California, hosted family BBQs, conquered The Chief, gone rock climbing and put our heads together to tackle creative challenges at our Hackathons)
*   We have an amazing existing team of talented developers, marketers, designers, customer champions, and other tech industry pros that want to work with you
*   Everyone has an equal opportunity to impact our product and company—we strongly value all of our people (the ones we work alongside and the ones we serve)
*   We are passionate about education—we provide learning opportunities and encourage each other to continuously learn and grow through workshops, training, blogs, meetups, books and mentorships
*   We provide a technology bonus for when you need to upgrade or replace your existing system
*   We care about helping you grow your career AND your family. Family is important to us, and our paid parental leave provides you with a little extra help while you’re beginning this new adventure.
*   We believe that different perspectives and backgrounds are what makes a company flourish. Thinkific welcomes all applicants regardless of race, gender, orientation, sexual identity, economic class, ability, disability, age, religious beliefs or disbeliefs, or status. We’re proud to be an inclusive company with values grounded in equality and ethics, where we embrace diversity. We welcome you!

## Requisitos - Requirements

*   Believe that better solutions are achieved through collaboration rather than independently
*   Are driven to continually improve by investing time in learning about new technologies or working on your own projects
*   Feel that “organized, with attention to detail” doesn’t even begin to describe you. You’ll have a lot of responsibility and also autonomy—no one will be looking over your shoulder to be sure you’ve got it right! A keen eye for detail is key in everything you do—quality over quantity is the name of the game.
*   Have previous Project Management or SCRUM experience
*   Demonstrate experience with build tools such as Webpack and npm scripts
*   Previously worked with and have experience with React.js and Ember.js
*   Have experience working with GraphQL API design
*   Are comfortable building Single Page Applications
*   Can effectively create, maintain and deploy applications to AWS - Cloud Computing Services

**Bonus points if you:**

*   Previously worked for a SaaS-based startup or growing technology company
*   Have a strong network of A-players to support our recruiting and people development efforts
*   Are comfortable working with container technology such as Docker and Kubernetes
*   Demonstrate experience with Golang
*   Have experience with Rails Engines

## Contratação - Job details

Full-time position in Vancouver, CA - sponsorship available

## Nossa empresa - About us

We’re about the results of online learning and the people along the way! Thinkific is a software platform that enables entrepreneurs to create, market, sell, and deliver their own online courses. Our mission is no less than to revolutionize the way people learn and earn online by giving them the tools they need to turn their expertise into a sustainable business that impacts both them and their audience. We believe in meaningful, innovative work: we're building and expanding an incredible product that empowers course creators around the globe, while working collaboratively to learn and succeed together. Our customer support is the best in the industry and one of the main reasons educators love using Thinkific!

## Como se candidatar - How to apply

Use the form in this link: http://bit.ly/thnk-eng-lead
**Multiple positions available** - Questions? Comment this issue in English

## Labels
Remoto
Exterior
Senior
